priam/src/main/java/com/netflix/priam/notification/BackupNotificationMgr.java

```diff
@@ -89,7 +94,7 @@ public void notify(BackupVerificationResult backupVerificationResult) {
     }
 
     private Map<String, MessageAttributeValue> getMessageAttributes(
-            AbstractBackupPath.BackupFileType backupFileType) {
+            AbstractBackupPath.BackupFileType backupFileType, JSONObject additionalAttributes) {
```

Comment:
We can leave this signature as simple as we found it considering the JSONObject you pass in will always have s3clustername and backuptype as keys. Please remove the BackupFileType input. and get the "always-on" s3clustername and backuptype out of the JSONObject you pass in.

Reply:
Done.